Documentation/requirements.md

```diff
@@ -0,0 +1,29 @@
+# Requirements
+
+## Ports
+
+The information below describes a minimum set of port allocations used by Kubernetes components.
+
+### Master node(s) ingress
+
+| Protocol | Port Range | Source                                    | Purpose                |
+-----------|------------|-------------------------------------------|------------------------|
+| TCP      | 443        | Worker Nodes, API Requests, and End-Users | Kubernetes API server. |
+| UDP      | 8472       | Master & Worker Nodes                     | flannel overlay network - *vxlan backend* |
```

Comment:
The healthchecks should be local from the kubelet on the same host-  so I don't think we would need to open them up to firewall rules external to the node.